backend/eventd/entity.go

```diff
@@ -20,39 +24,86 @@ func createProxyEntity(event *corev2.Event, s store.EntityStore) error {
 	}
 
 	// Determine if the entity exists
-	ctx := context.WithValue(context.Background(), corev2.NamespaceKey, event.Entity.Namespace)
-	entity, err := s.GetEntityByName(ctx, entityName)
-	if err != nil {
-		return err
-	}
+	//NOTE(ccressent): there is no timeout for this operation?
```

Comment:
There is an implicit timeout on the store governed by the etcd client, but no explicit timeout for `dead` callbacks. Perhaps we should consider adding one? It's unclear to me if it would help or harm.

Reply:
Not sure either, I originally left that note because similar operations in `dead()` do have a timeout in their context: https://github.com/sensu/sensu-go/blob/524efa49a13840381dc2a12b2ef0d463f0af462c/backend/eventd/eventd.go#L348-L352